bpython/cli.py

```diff
@@ -160,6 +161,7 @@ def __init__(self, interface):
         self.encoding = getpreferredencoding()
         self.interface = interface
         self.buffer = list()
+        self.errors = "strict"
```

Comment:
I forget now... I actually made these original changes a long time ago. I think was related to getting it to work with Python 3. It could probably be removed, but I kept it just in case there was still a reason for it.